### ISSUE TITLE
[Warrior] Recklessness and Slam

### ISSUE THREAD

**Issue by AlexCSCP** [dalaranwow/dalaran-wow#5450]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When you use Slam under the effect of Recklessness (having at least 1 charge), you can do normal (yellow, not crit, not glancing, not blocked/parried/dodged/missed) strikes.
It doesn't seems to increase Slam critical chance but it does consume charges if slam crit.

**Expected behaviour**: Tell us what should happen instead
Recklessness add 100% critical chance to the next 3 ability attacks, slam being a normal melee ability, should be affected by it.
In the case I'm mistaken and, for some obscure reason, it should not be affected by Recklessness, then it should not consume charges.

**Steps to reproduce the problem**:

1. Get rid of all +crit items (to get low crit chance).
2. Find a mob with the same or lower lvl (to avoid problems with the target defense).
3. Use Recklessness and spam Slam.

**Include proofs for this behaviour**
Saw it happening on a dungeon and went to test it on a normal, same lvl, mob.
With less than 20% of crit chance I could get non crits with slam (a screenshot at the bottom).

Info:
https://wowgaming.altervista.org/aowow/?spell=1719#modifies:50+1+13+3
Slam is  in the list of "Modifies".
http://wowwiki.wikia.com/wiki/Recklessness?oldid=2297223

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
1719

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
https://i.imgur.com/WJFZLHz.jpg
You can see in the combat log the "normal" slams, between the start and end of Recklessness.

**Comment by AlexCSCP** [comment]
To be sure, I tried with heroic strike on same mob and I did get only critical strikes, so it's not a problem of the mob or my character.

**Comment by Aridam** [comment]
I did Recklessness + Slam 3x and all three Slams were Crits.

Make sure when you do Recklessness, you don't do any AoE-/Cleave-based Damage which may consume Recklessness charges -- Cleave, Whirlwind and so on -- they bug Recklessness out and get it stuck, being unable to consume any more Procs, while not landing Crits with it, ending-up stuck with it and having to do /cancelaura with it else you're just taking more Damage with no trade-off at that point.

https://github.com/dalaranwow/dalaran-wow/issues/542

**Comment by Rushor** [comment]
moved to https://github.com/dalaranwow/dalaran-wow/issues/542

**Comment by AlexCSCP** [comment]
@Aridam 
Look at the screenshot (https://i.imgur.com/WJFZLHz.jpg), slam is done alone with no AoE nor Autoattack, and still doesn't crit.  So your theory of Recklessness being bugged by other skills is irrelevant when it comes to slam.

**Comment by Aridam** [comment]
Check the moved report with the master-report for my response.
